client/src/app/core/auth/auth.service.ts

```diff
@@ -240,6 +243,48 @@ Ensure you have correctly configured PeerTube (config/ directory), in particular
         )
   }
 
+  getScopedTokens (): Promise<ScopedToken> {
```

Comment:
Create a dedicated service for scoped tokens and use Observable pattern for consistency with other HTTP service methods